pywt/tests/test_multilevel.py

```diff
@@ -50,6 +52,56 @@ def test_swt_decomposition():
     assert_(pywt.swt_max_level(len(x)) == 3)
 
 
+def test_swt_iswt_integration():
+    """
+    This function performs a round-trip swt/iswt transform test on
+    all available types of wavelets in PyWavelets - except the 'dmey'
+    wavelet. The latter has been excluded because it does not produce
+    very precise results. This is likely due to the fact that the
+    'dmey' wavelet is a discrete approximation of a continuous
+    wavelet. All wavelets are tested up to 3 levels. The test
+    validates neither swt or iswt as such, but it does ensure
+    that they are each other's inverse.
+    """
+    max_level = 3
+    wavelets = pywt.wavelist()
+    if 'dmey' in wavelets:
+        wavelets.remove('dmey')  # The 'dmey' wavelet seems to be a bit special - disregard it for now
+    for current_wavelet_str in wavelets:
+        current_wavelet = pywt.Wavelet(current_wavelet_str)
+        input_length_power = np.ceil(np.log2(max(current_wavelet.dec_len, current_wavelet.rec_len)))
```

Comment:
style nitpick: maybe break this line for readability, ``current_wavelet.rec_len)))`` lined up with ``current_wavelet.dec_len,``. 
And move the ``int``coercion from the line below to this line, because ``input_length_power`` is meant to be integer.